src/main/resources/configCivcraft.yml

```diff
@@ -7216,7 +7216,7 @@ recipes:
         durability: 3
       eggs:
         material: EGG
-        amount: 8
+        amount: 4
```

Comment:
I know you guys did your best to make every recipe feel unique. But I honestly feel like vines are a really good ingredient that could be use to replace some of the harder and annoying parts of topaz and amethyst. I don't really know two other unique ingredients that fit the same bill.

Vines grow everywhere, they can be afked, but it takes effort to build an afk farm. It also takes a decent amount of time to grow enough. My best suggestion right now is to replace eggs in topaz and ink sacks in amethyst with vines. Keep the apples in amethyst the same for now, as while its annoying they are at least an easy farm to make. I'll modify the pull request now.